<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>10</y>
      <width>446</width>
      <height>286</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="generator_label">
           <property name="text">
            <string>Генератор заявок:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="a_label">
               <property name="text">
                <string>A =</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="a_spin">
               <property name="minimum">
                <double>-100000000000000003502199685943161173046080317798311825604870144.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999999999999992486776161899288204254467086983483846143922597222529419997579302660316349376281765375153005841365553228283904.000000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="b_lable">
               <property name="text">
                <string>B =</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="b_spin">
               <property name="minimum">
                <double>-100000000000000003502199685943161173046080317798311825604870144.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999999999999992486776161899288204254467086983483846143922597222529419997579302660316349376281765375153005841365553228283904.000000000000000</double>
               </property>
               <property name="value">
                <double>2.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="service_label">
           <property name="text">
            <string>Обслуживающий аппарат:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="mu_label">
               <property name="text">
                <string>mu =</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="mu_spin">
               <property name="minimum">
                <double>-100000000000000003502199685943161173046080317798311825604870144.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999999999999992486776161899288204254467086983483846143922597222529419997579302660316349376281765375153005841365553228283904.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="sigma_label">
               <property name="text">
                <string>sigma =</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="sigma_spin">
               <property name="minimum">
                <double>-100000000000000003502199685943161173046080317798311825604870144.000000000000000</double>
               </property>
               <property name="maximum">
                <double>99999999999999992486776161899288204254467086983483846143922597222529419997579302660316349376281765375153005841365553228283904.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="return_label">
         <property name="text">
          <string>Вероятность возврата в очередь =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="return_spin">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QRadioButton" name="dt_radio">
         <property name="text">
          <string>dt метод</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="event_radio">
         <property name="text">
          <string>событийный метод</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="dt_label">
           <property name="text">
            <string>dt =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dt_spin">
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="stats_label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>t сбора статистики =</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="stats_spin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="requestcount_radio">
         <property name="text">
          <string>Общее количество обработанных заявок:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="requestcount_spin"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="time_radio">
         <property name="text">
          <string>Время работы системы:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="time_spin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="run_button">
       <property name="text">
        <string>Вычислить оптимальную длину очереди</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="optimal_label">
       <property name="text">
        <string>Оптимальная длина очереди:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
